### PR TITLE
Root page design updates and internationalization

### PR DESCRIPTION
**Pull request by nonoakij**
## Checklist
- [ ] [versioned](https://github.com/changesets/changesets/blob/main/docs/command-line-options.md#version) properly

### JP
Theme| Mobile | Tablet
-|-|-
Light|localhost_8000_ (1)|localhost_8000_ (2)
Dark|localhost_8000_|localhost_8000_ (3)

Dark|
### EN
Theme| Mobile | Tablet
-|-|-
Light|localhost_8000_en_ (2)|localhost_8000_en_ (1)
Dark|localhost_8000_en_ (3)|localhost_8000_en_

### CODE REVIEW

File: packages/app/src/components/button/filled/index.tsx [hunk 1]
```diff
@@ -89,4 +89,62 @@ const FilledButton = function <T = null>({
     </button>
   );
 };
+
+const Renewal = function <T = null>({
```
Comment (nonoakij) [inline packages/app/src/components/button/filled/index.tsx:93]: The component named Renewal will be offered this way until all design updates are complete.

When all changes are complete, replace the original component with the Renewal component.

**Comment by h8570rg** [comment]
github icon and twitter icon are swapped!

File: packages/app/src/components/navigation/links/index.tsx [hunk 2]
```diff
@@ -38,7 +38,9 @@ const links: {
 ];
 
 type Props = BaseProps;
-const Links: React.FC<Props> = ({ className = '', on }) => {
+const Links: React.FC<Props> & {
+  renewal: React.FC<Props>;
```
Comment (h8570rg) [inline packages/app/src/components/navigation/links/index.tsx:42]: What is this used for?

**Comment by nonoakij** [comment]
> github icon and twitter icon are swapped!

This is an intentional modification to the design update.
The order in which the "services" are displayed depends on this array.

**Comment by h8570rg** [comment]
Is this correct? I found it on screenShot.
image

File: packages/app/src/components/navigation/links/index.tsx (hunk 2 above)
Comment (nonoakij) [reply packages/app/src/components/navigation/links/index.tsx:42]: This is for using the component like this:
```tsx
<Links />
<Links.renewal />
```

**Comment by nonoakij** [comment]
> Is this correct? I found it on screenShot. image

Oops, this is my mistake. I will fix it now.
Thanks.

File: packages/app/src/components/navigation/links/index.tsx (hunk 2 above)
Comment (h8570rg) [reply packages/app/src/components/navigation/links/index.tsx:42]: Thanks!

**Comment by nonoakij** [comment]
@h8570rg I fixed it.
c2251b91c5c5de8e86f46cf9396a9d909899dc12

**Review by h8570rg — APPROVED** [review]

**Review by ejithon — APPROVED** [review]

**Pull request merged**